components/ws-manager-bridge/src/cluster-service-server.ts

```diff
@@ -357,6 +361,20 @@ function mapToGRPCError(err: any): any {
     return err;
 }
 
+function getUserAgentAndClientName(call: grpc.ServerUnaryCall<any,any>): [string,string] {
+    const clientNameHeader = call.metadata.get("client-name")
+    const userAgentHeader = call.metadata.get("user-agent")
+
+    let [clientName, userAgent] = ["", ""]
+    if (clientNameHeader.length != 0) {
+        clientName = clientNameHeader[0].toString()
+    }
+    if (userAgentHeader.length != 0) {
+        userAgent = userAgentHeader[0].toString()
+    }
+    return [clientName, userAgent]
```

Comment:
If you return a shape ` { clientName, userAgent }` that's in line with all other usages of `log.info`, and we make sure we can properly filter it in GCloud, for instance.